diagnostic.md

```diff
@@ -51,13 +69,18 @@ Write the `curl` request you'd use to delete a single donut, using whatever ID
 you'd like.
 
 ```sh
-# your answer here
+curl --include --request DELETE 'http://www.example.com/donuts/10'
 ```
 
 Write an AJAX request to delete a single donut from the server.
 
 ```js
-let deleteDonut = /* your answer here */;
+let deleteDonut = function (event){
+  event.preventDefault();
+  libraryApi.destroy(event.target)
+    .done(ui.logResponseBody)
+    .fail(ui.logRequestError);
+};
```

Comment:
1/2. I see what you wanted to do here but this is not an ajax request, the libraryApu.destroy() function that this calls would be the ajax request